src/v1/waittimes/routes.ts

```diff
@@ -0,0 +1,46 @@
+import * as express from 'express';
+import { Redis } from 'ioredis';
+import { WaitTimeDB } from './db';
+import asyncify from '../lib/asyncify';
+import { cache } from '../lib/cache';
+import { generateKey } from '../server';
+
+import bodyParser = require("body-parser");
+
+export default function routes(redis?: Redis, credentials?) {
+  const db = new WaitTimeDB();
+
+  const router = express.Router();
+  router.use(bodyParser.json());
+
+  const key = (req) => generateKey(req, "/waitTime", ["id"]);
+  router.get(
+    "/waitTime",
+    cache(key, redis),
+    asyncify(async (req, res) => {
+      try {
+        const query = await (req.query.id
+          ? db.waitTime(req.query.id)
+          : db.waitTime());
```

Comment:
The way `db.waitTime` is written, can't this ternary expression be simplified to `db.waitTime(req.query.id)`? Because the check occurs in the function.

Reply:
Yeah, that makes sense. Thanks, I'll do that!